stable/kapacitor/Chart.yaml

```diff
@@ -1,5 +1,6 @@
 name: kapacitor
-version: 0.5.0
+version: 0.5.1
+appVersion: 0.5.1
```

Comment:
looking at https://github.com/AdamDang/charts/blob/e28a7a22a5d2af94f25a58ac64f7eecb76dde481/stable/kapacitor/values.yaml#L6, it appears like appVersion should be `1.4`

Reply:
@sameersbn 
Thanks! I've modified the appVersion to 1.4-alpha.